Dropdown/Dropdown.js

```diff
@@ -35,17 +38,26 @@ import compose from 'ramda/src/compose';
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import Button from '../Button';
+import {ButtonBase} from '../Button';
 import Icon from '../Icon';
 import Item from '../Item';
 import RadioItem from '../RadioItem';
 import Scroller from '../Scroller';
 import Skinnable from '../Skinnable';
+import TooltipDecorator from '../TooltipDecorator';
 
 import componentCss from './Dropdown.module.less';
 
 const oppositeDirection = {left: 'right', right: 'left', up: 'down', down: 'up'};
 const ContainerDiv = SpotlightContainerDecorator({enterTo: 'last-focused'}, 'div');
+const ButtonDecorator = compose(
+	Pure,
+	UiButtonDecorator,
+	TooltipDecorator({tooltipDestinationProp: 'decoration'}),
+	Spottable,
+	Skinnable
+);
+const Button = ButtonDecorator(ButtonBase);
 const MarqueeButton = MarqueeDecorator({className: componentCss.marquee}, Button);
```

Comment:
I think there were two choices.

Can you explain why you are remove the marquee from the Button, not line 49? (Please leave on the description)

Because we are in remote, so please explain in detail.

Reply:
On line 49, a class `marquee` is applied. it's role is to prevent the overlap of the marquee text with the arrow down/up icon on its' right. That's why I chosed to remove the marquee which comes from agate/Button.

If I were to choose the other one, some adjustments are needed on agate/Button. (add there a marquee class when applying MarqueeDecorator, then set flex-grow:1 for marquee and text-align:center)

Which approach would you prefer?

Reply:
I don't have any preference. if there is a reason, and an internal review decided that this direction is good, it's okay.

Reply:
The second approach sounds cleaner. I would opt for that.

Reply:
Actual approach looks good to me.